ha/cli/exec/clusterExecutor.py

```diff
@@ -134,11 +134,51 @@ def execute(self) -> None:
         Log.info("Cluster started successfully")
 
 class ClusterStopExecutor(CommandExecutor):
+
+    def __init__(self):
+        '''Init method'''
+        super(ClusterStopExecutor, self).__init__()
+        self._pcs_cluster_controller = PcsClusterController()
```

Comment:
Should not get PcsClusterController directly. Please change if it was done in earlier PR too.

Reply:
Example:
----------------------
self._cluster_manager = CortxClusterManager()

for interface check
https://github.com/Seagate/cortx-ha/blob/main/ha/test/integration/cluster_manager/test_cm.py

Reply:
Done.